apisix/plugins/kafka-logger.lua

```diff
@@ -74,6 +74,15 @@ local schema = {
         inactive_timeout = {type = "integer", minimum = 1, default = 5},
         batch_max_size = {type = "integer", minimum = 1, default = 1000},
         include_req_body = {type = "boolean", default = false},
+        include_req_body_expr = {
```

Comment:
Before we can merge it, could you add a check for the expr in the `check_schema`? Like:
https://github.com/apache/apisix/blob/9b355c6f13194c00eeb491542ee0727467988877/apisix/plugins/response-rewrite.lua#L111

Reply:
done